### ISSUE TITLE
Got error when run in dev environment

### ISSUE THREAD

**Issue by billsliu** [Clariity/react-chessboard#121]
**I got following error in Chrome inspection tool when run this chessboard in dev environment.**

You are currently using minified code outside of NODE_ENV === "production". This means that you are running a slower development build of Redux. You can use loose-envify (https://github.com/zertosh/loose-envify) for browserify or setting mode to production in webpack (https://webpack.js.org/concepts/mode/) to ensure you have the correct code for your production build.
![2023-12-27 (1)](https://github.com/Clariity/react-chessboard/assets/5201301/f614eb57-90ff-49ed-ace9-0a31e70dff73)

**Steps to reproduce the error.**
**Step 1 — Create a blank React App**
npx create-next-app@latest
**Step 2 — Install dependencies**
npm i react-chessboard
**Step 3 — Add react-chessboard component**
Change pages/index.js to be the following:

```
import { Chessboard } from "react-chessboard"

export default function Home() {
  return (
    <Chessboard position={"start"} />
  )
}
```

**Comment by billsliu** [comment]
Got this error when I use npm run dev in development, this error will not show when use npm run build && npm run start.

I use more than 20 package in my project, but only react-chessboard has this issue, I feel this package doesn't follow the standard rule to create the component.

**Comment by Clariity** [comment]
Fix applied in 4.3.4
